.github/workflows/test.yaml

```
@@ -113,7 +113,7 @@ jobs:
           # by GitHub. The availability of the Python version
           # depends on the version of Ubuntu in use. 
           # See https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
-          - python-version: '3.8.12'
+          - python-version: '3.9.22'
```

I think that we can use the oldest version availabe on GitHub ... I seee 3.9.0 is available

```suggestion
          - python-version: '3.9.0'
```

---

I ultimately had to change this to "3.9" to get it to build.